nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ControlRate.java

```diff
@@ -115,6 +115,13 @@
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .expressionLanguageSupported(false)
             .build();
+    public static final PropertyDescriptor MAX_FF_PER_TRIGGER = new PropertyDescriptor.Builder()
```

Comment:
@jskora I'm not sure that this needs to be configurable. This is an implementation detail that feels a bit leaky to me. Users do not know what an 'onTrigger() call' is. We should probably just cap it at say 1000 and not more than the max number of FlowFiles to transfer per 'Time Duration'.